### ISSUE TITLE
facade-x IRI not compliant with IETF scheme

### ISSUE THREAD

**Issue by enridaga** [SPARQL-Anything/sparql.anything#28]
Jena complains as follows:
```
Bad IRI: <facade-x:media-type=text/html,html.selector=#az-group,location=https://imma.ie/artists/> Code: 45/UNREGISTERED_NONIETF_SCHEME_TREE in SCHEME: The scheme name has a "-" in it, but it does not start in "x-" and the prefix is not known as the prefix of an alternative tree for URI schemes.
```
We may change the URI scheme, discussion open in alternatives.

**Comment by enridaga** [comment]
x-sparql-anything:

**Comment by luigi-asprino** [comment]
Closed with commit bfafebc
